lib/components/MapView.js

```diff
@@ -915,7 +937,7 @@ if (Platform.OS === 'android') {
 }
 const getAirMapComponent = provider => airMaps[provider || 'default'];
 
-const AIRMapLite = NativeModules.UIManager.AIRMapLite &&
+const AIRMapLite = NativeModules.UIManager.getViewManagerConfig('AIRMapLite') &&
```

Comment:
This probably will crash on RN <0.58 because UIManager does not have property getViewManagerConfig. Right?

Reply:
@vovkasm Good point, I just pushed a commit to add a quick check for `UIManager.getViewManagerConfig` as with the other changes.